config.yml.example

```diff
@@ -16,6 +16,56 @@ api:
   auth: ""
   path: ""
 
+users_api:
+  auth: ""
+  path: ""

```

Comment:
Might be worth to change that parameter name to url, as its used as a URL and not as a path.